charts/alfresco-repository/ci/kustomize/kustomization.yaml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
I am not a fan of this approach because is already causing issues in the sync chart because nothing is going to wipe the resources installed by this chart when the test finish and that's a problem when testing more than one chart and due to runners limited resources

Reply:
To be honest I like the optional dependecies even less for the reason explained below.
Though  I admitedly didn't take the reosource issue into account

Reply:
Tracked in https://alfresco.atlassian.net/browse/OPSEXP-2216 for now